pytensor/tensor/shape.py

```diff
@@ -994,6 +995,9 @@ def specify_broadcastable(x, *axes):
     if not axes:
         return x
 
+    if min(axes) < 0:
```

Comment:
No need to check, we can always run. Also the check below is no longer needed as the normalize helper raises if an axis is invalid